src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -11,43 +20,32 @@
 
 public class ArmSubsystem extends SubsystemBase implements Loggable {
     public enum State {
-        Pickup, Rest, PlacingA, PlacingB, PlacingC, PlacingD, PlacingE, Transit // Possible System States
-    }
-
-    private final double elbowPosToDegrees = 599.9;
-    private final double shoulderPosToDegrees = 342.8;
-    private final double stateTolerance = 2;
-
-    // System motor controllers
-    private final CANSparkMax m_shoulder = new CANSparkMax(ArmConstants.SHOULDER_ID, MotorType.kBrushless);
-    private final CANSparkMax m_shoulderFollower = new CANSparkMax(ArmConstants.SHOULDER_FOLLOWER_ID, MotorType.kBrushless);
-    private final CANSparkMax m_elbow = new CANSparkMax(ArmConstants.ELBOW_ID, MotorType.kBrushless);
-
-    // System State - init in rest state
-    private State m_state_desired = State.Rest;
-    private State m_state_actual = State.Rest;
+        Pickup(0, 0), Rest(0, 0), PlacingA(Math.PI / 4, 0), PlacingB(0, 0), PlacingC(0,
+                Math.PI / 2), PlacingD(Math.PI / 2, Math.PI / 2), PlacingE(0, 0);
 
-    private final NetworkTableInstance handle = NetworkTableInstance.getDefault();
+        public final double elbow;
+        public final double shoulder;
 
-    public void printState() {
-        System.out.print("Desired: ");
-        System.out.print(this.m_state_desired);
-        System.out.print("\t\t Actual: ");
-        System.out.print(this.m_state_actual);
-    }
-
-    // Class to hold position Data for each system state
-    private static class POS {
-        public int elbow, shoulder = 0;
-        public POS(int shoulder, int elbow) {
+        private State(double elbow, double shoulder) {
             this.elbow = elbow;
             this.shoulder = shoulder;
         }
     }
 
+    private State m_desiredState = State.Rest;
+
+    // System motor controllers
+    private final CANSparkMax m_shoulder =
+            new CANSparkMax(ArmConstants.SHOULDER_ID, MotorType.kBrushless);
+    private final CANSparkMax m_shoulderFollower =
+            new CANSparkMax(ArmConstants.SHOULDER_FOLLOWER_ID, MotorType.kBrushless);
+    private final CANSparkMax m_elbow =
+            new CANSparkMax(ArmConstants.ELBOW_ID, MotorType.kBrushless);
+
     private static class PID {

```

Comment:
Where is the rest of this class?

Reply:
Which class?

Reply:
I misspoke, the constructor that includes all 5 parameters.

Reply:
Sorry... which 5 params?